webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -234,19 +234,17 @@ class ReportDisplayer(
 
         case R.KeepLastCompliance =>
           (
-            <p>No recent reports have been received for this node in the grace period since the last configuration policy change.
-              The grace period is expired since {displayDateOpt(info.expiredSince)}.
-               Still, this node has a configuration to keep its last known compliance until {
-              displayDateOpt(info.expirationDateTime)
-            }.</p><p>{
-              info.lastRunDateTime match {
-                case None    =>
-                  "We don't have more information about a run that might have provided compliance."
-                case Some(t) =>
-                  s"The run from which that compliance was computed started at ${displayDate(t)}."
+            <p>This node is in "keep compliance" mode because it didn't received any reports recently.</p>
+            <p>The last reports received are from 🕔 <b>{displayDateOpt(info.expiredSince)}</b>.</p>
+            <p>That mode will expire if no reports are received before {displayDateOpt(info.expirationDateTime)}.</p> ++ {
+              info.lastRunConfigId match {
+                case Some(id) =>
+                  <p>The last compliance was for configuration ID '{id.value}' for a run executed at {
+                    displayDateOpt(info.expirationDateTime)
+                  }</p>
```

Comment:
Shouldn't it be the `info.lastRunDateTime` here instead of `info.expirationDateTime` ?

Reply:
yes